docs/advanced_topics/images/image_formats.rst

```diff
@@ -0,0 +1,49 @@
+Image formats
+=========================
+
+Using the picture element
+-------------------------
+
+The `picture element <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture>`_
+can be used with wagtails ``format-<type>`` image operation to specify different
+image formats and let the browser choose it's preferred one.
```

Comment:
"it's" → "its"

But more generally, "[..] let the browser choose the one it prefers" reads a bit better